gluoncv/model_zoo/mask_rcnn/mask_rcnn.py

```diff
@@ -155,6 +155,30 @@ def reset_class(self, classes, reuse_weights=None):
                     # set data to new conv layers
                     new_params.set_data(new_data)
 
+class MRCNNMaskTargetWrapper(nn.HybridBlock):
```

Comment:
Could u refactor this into MaskTargetGenerator? since they do the same thing

Reply:
Done